docs/Build.md

```diff
@@ -102,7 +102,7 @@ Webpack allows file specific loaders or utilities to be invoked as a part of the
 | Rule                        | Plugin/loader(s)                                                                                        |                                                                                                                                                                                                   Purpose |
 | --------------------------- | ------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
 | `/(\.css\|.scss)$/`         | `style-loader` (dev only), `miniCssExtractPlugin.loader` (production only), `css-loader`, `sass-loader` | Handle scss/css loading/references. If dev mode, use `style-loader` for speed, else use `miniCssExtractPlugin.loader` (in combination with the `miniCssExtractPlugin` plugin) to produce/emit css file(s) |
-| `/(\.ts|tsx)$/`             | `ts-loader`                                                                                             |                                                                                                                                                               Transpiles TypeScript files into JavaScript |
+| `/(\.ts                     | tsx)\$/`                                                                                                |                                                                                                                                                                                               `ts-loader` | Transpiles TypeScript files into JavaScript |
```

Comment:
This additional bar is breaking the table formatting. Did you add it (or did the linter)?

Reply:
Same issue seen in #66 - I am at a loss to why this is an issue (and not happened before)

Reply:
It happened due to issue in `npm lint`

Reply:
I have just seen that the `|` between ts and tsx is not escaped (like css/scss above). Hence the linter (for whatever reason now) is kicking in an treating it as a markdown column. As a part of this change, can the `\` be added?

Reply:
For completeness, I am at a loss when I changed this file yesterday and did not have any of these issues!

Reply:
Added skip character (like css/scss) between ts and tsx